src/libcore/ptr.rs

```diff
@@ -2521,6 +2545,7 @@ impl<T: ?Sized> Shared<T> {
     /// The resulting lifetime is bound to self so this behaves "as if"
     /// it were actually an instance of T that is getting borrowed. If a longer
     /// (unbound) lifetime is needed, use `&*my_ptr.ptr()`.
```

Comment:
Should this be `&*my_ptr.as_ptr()`?

Reply:
Good catch! Fixed. (This was a pre-existing bug.)